third_party/lychee/repos.bzl

```diff
@@ -0,0 +1,22 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:repo.bzl", "http_archive_or_local")
+
+def lychee_repos(local = None):
+    LYCHEE_VERSION="v0.11.1"
```

Comment:
@HU90m Looks like v0.12.0 came out two weeks ago on <https://github.com/lycheeverse/lychee/releases>. Should we upgrade to that?